### PR TITLE
fix: dolibarr security alerts

### PR DESCRIPTION
**Pull request by Jemy64**
- disable php functions
- change directory and files permissions

### CODE REVIEW

File: images/19.0.0-php8.2/Dockerfile [hunk 1]
```diff
@@ -40,26 +40,26 @@ ENV PHP_INI_POST_MAX_SIZE 8M
...
+    libjpeg62-turbo-dev \
+    libkrb5-dev \
+    libldap2-dev \
+    libpng-dev \
+    libpq-dev \
+    libxml2-dev \
+    libzip-dev \
+    default-mysql-client \
+    postgresql-client \
+    cron \
```
Comment (tuxgasy) [inline images/19.0.0-php8.2/Dockerfile:62]: Why add/remove spaces here ?

File: images/19.0.0-php8.2/Dockerfile [hunk 2]
```diff
@@ -80,8 +80,13 @@ RUN curl -fLSs https://github.com/Dolibarr/dolibarr/archive/${DOLI_VERSION}.tar.
     cp -r /tmp/dolibarr-${DOLI_VERSION}/scripts /var/www/ && \
     rm -rf /tmp/* && \
     mkdir -p /var/www/documents && \
+    chmod -R 644 /var/www/documents && \
     mkdir -p /var/www/html/custom && \
-    chown -R www-data:www-data /var/www
+    chown -R www-data:www-data /var/www && \
+    chmod -R 444 /var/www/html/* && \
+    chmod 774 /var/www/html/ && \
```
Comment (tuxgasy) [inline images/19.0.0-php8.2/Dockerfile:87]: it's better to put 775, no ?

File: images/19.0.0-php8.2/docker-run.sh [hunk 3]
```diff
@@ -46,7 +46,7 @@ upload_max_filesize = ${PHP_INI_UPLOAD_MAX_FILESIZE}
 post_max_size = ${PHP_INI_POST_MAX_SIZE}
 allow_url_fopen = ${PHP_INI_ALLOW_URL_FOPEN}
 session.use_strict_mode = 1
-disable_functions = pcntl_alarm,pcntl_fork,pcntl_waitpid,pcntl_wait,pcntl_wifexited,pcntl_wifstopped,pcntl_wifsignaled,pcntl_wifcontinued,pcntl_wexitstatus,pcntl_wtermsig,pcntl_wstopsig,pcntl_signal,pcntl_signal_get_handler,pcntl_signal_dispatch,pcntl_get_last_error,pcntl_strerror,pcntl_sigprocmask,pcntl_sigwaitinfo,pcntl_sigtimedwait,pcntl_exec,pcntl_getpriority,pcntl_setpriority,pcntl_async_signals,passthru,shell_exec,system,proc_open,popen
+disable_functions = pcntl_alarm,pcntl_fork,pcntl_waitpid,pcntl_wait,pcntl_wifexited,pcntl_wifstopped,pcntl_wifsignaled,pcntl_wifcontinued,pcntl_wexitstatus,pcntl_wtermsig,pcntl_wstopsig,pcntl_signal,pcntl_signal_get_handler,pcntl_signal_dispatch,pcntl_get_last_error,pcntl_strerror,pcntl_sigprocmask,pcntl_sigwaitinfo,pcntl_sigtimedwait,pcntl_exec,pcntl_getpriority,pcntl_setpriority,pcntl_async_signals,passthru,shell_exec,system,proc_open,popen,dl,apache_note,apache_setenv,show_source,virtual
```
Comment (tuxgasy) [inline images/19.0.0-php8.2/docker-run.sh:49]: May I know why it is necessary to deactivate these functions?

**Review by tuxgasy — COMMENTED** [review]
These changes must be done in the root files (see [CONTRIBUTING.md](https://github.com/tuxgasy/docker-dolibarr/blob/master/CONTRIBUTING.md))

File: images/19.0.0-php8.2/Dockerfile (hunk 1 above)
Comment (Jemy64) [reply images/19.0.0-php8.2/Dockerfile:62]: This is not voluntary but the default formatting of vscode. I will revert this change.

File: images/19.0.0-php8.2/docker-run.sh (hunk 3 above)
Comment (Jemy64) [reply images/19.0.0-php8.2/docker-run.sh:49]: These functions make it possible to perform hot changes on the system. Dolibarr recommends disabling them in the security menu.

**Review by tuxgasy — APPROVED** [review]

**Comment by tuxgasy** [comment]
You have to run update.sh script.

**Pull request closed without merging**